### ISSUE TITLE
Site fails to load, says HttpsGuard in tab label

### ISSUE THREAD

**Issue by ez2remember** [EFForg/https-everywhere#5126]
Kept having trouble with similar links from cvent, although cvent.com opens fine.

http://www.cvent.com/events/the-destiny-of-america/event-summary-85cbf4bc57284dfca5a160e7b3abd7dc.aspx

I tried on Internet Explorer and had the same issue. I disabled Https Everywhere extension in Chrome, and then it seemed to work.

**Comment by J0WI** [comment]
Evil JS redirect:
```
var httpsLocation = 'http://' + document.location.host + document.location.pathname + document.location.search;
document.location = httpsLocation;
```

**Comment by Bisaloo** [comment]
I fixed this in #14335.
